doc/upgrading.md

```diff
@@ -13,6 +13,9 @@ See the [NEWS](../NEWS.md) summary of changes for each version.
 
 ## Version 29
 
+* RMB-510 Tenant POST `/_/tenant` must include a body. Older versions of RMB
+  made a special case for zero length body in which case the JSON content
+  was not decoded.
```

Comment:
The second sentence should be removed, the upgrading information is about *how* to upgrade a module. Information about *why* the upgrade is needed is off-topic in this upgrade recipe and can be looked up via the issue number; you may convert the ticket number into a clickable link `[RMB-510](https://issues.folio.org/browse/RMB-510)`, though.
Adding an example for a minimal JSON body may speed up the upgrading.